src/Components/Calendar/EndDate.tsx

```diff
@@ -1,54 +1,61 @@
 // react-datepicker를 사용해서 마감날짜 구현 328px, 44px, ex) 24.01.23
-import DatePicker from 'react-datepicker';
-import { useState } from 'react';
+import React, { useState } from 'react';
+import { ControllerRenderProps } from 'react-hook-form';
+
+// date-picker
+import DatePicker, { ReactDatePicker } from 'react-datepicker';
+import { registerLocale } from 'react-datepicker';
+import ko from 'date-fns/locale/ko';
 import 'react-datepicker/dist/react-datepicker.css';
+import { parseISOString } from '@/utils/date';
 import styled from 'styled-components';
-import { OptionalCreates } from '@/Pages/Studies/CreateRecruitment';
-import { Creates } from '@/Types/studies';
-
-export type Props = {
-  onClick?: () => void;
-  children?: React.ReactNode;
-  // onChange?: (event: string) => void;
-  setForm: (any: OptionalCreates) => void;
-  useForm: Creates;
-  value?: string;
-  type?: string;
-  name?: string;
-  maxlength?: number;
-  id?: string;
-  formData?: number | string;
-  ref?: string;
-};
-
-export const EndDate = ({ useForm }: Props) => {
-  const [startDateTime, setForms] = useState(new Date());
-
-  return (
-    <DateContainer
-      value={(useForm.recruitmentEndDateTime = startDateTime.toISOString().slice(0, -5))}
-      selected={startDateTime}
-      dateFormat="yy.MM.dd"
-      onChange={(date: Date) => setForms(date)}
-      placeholderText="ex)24.01.07"
-      isClearable={true}
-    />
-  );
-};
+
+registerLocale('ko', ko);
+
+interface IFormValues {
+  recruitmentEndDateTime: string;
+}
+
+interface Props {
+  defaultValue?: string;
+}
+
+export const EndDate = React.forwardRef<ReactDatePicker<string, boolean>, ControllerRenderProps<IFormValues> & Props>(
+  ({ onChange, name, defaultValue }, ref) => {
+    const today = new Date(Date.now());
```

Comment:
```suggestion
    const today = new Date();
```

`new Date()`로 새 Date 객체를 생성할 때, 인자가 없으면 기본적으로 현재 시간(now을) 기준으로 생성합니다.
따라서 위 코드를 정확히 같은 내용을 객체를 두 번씩 생성하며 반복하게 됩니다.

참고: 
https://stackoverflow.com/a/66925958/16246435

Reply:
(실수로 comment로 올려서 지웠다가 다시 올렸습니다)

Reply:
말씀하신대로 기본값이 현재 시간을 기준으로 생성되니까 인자를 제거해도 될 것 같습니다. 수정했습니다!